example/axisnow/decryptAndEmbed.ts

```diff
@@ -0,0 +1,126 @@
+import Decryptor from '@nypl-simplified-packages/axisnow-access-control-web';
```

Comment:
We should make sure that any users know that this is a specific decryption scheme, and that it's not supposed to work unless you're working with very specific books.  

(and that to any member of the public that's trying to use this repo, they should ignore this)

Reply:
I think some of this file should be extracted to be an interface.  Eg:  a `decryptChapter` and `embedImageAssets` function should be available regardless of which Decryptor you choose?  

However, it's up to you to actually implement your decryptor

Reply:
Yea we could do that although I sort of doubt anyone will need client side decryption initially, and if they do idk that our methods will work for them, or if their setup would be significantly different ya know? Like I could make it so the app takes a: `type Decryptor = (in: Uint8Array) => Promise<Uint8Array>`... and then we would call these `decryptChapter` and `embedImageAssets` internally. I'm just dubious whether anyone else's decryptor would necessarily function similarly enough to make that useful. It also wouldn't work on PDF content. So I guess I was more inclined to just say "hey, here's a way to define a function that takes the resourceUrl and returns a string. Do whatever you need to do in there"